developer/python-django-devtools-reference.rst

```diff
@@ -0,0 +1,244 @@
+Python/django dev tools reference
+=================================
+
+In this page, we aim at helping technical users and contributors to work
+on OpenWISP and we introduce to some python tools they can find very
```

Comment:
I would write `we introduce them to some python tools they might find very useful`.